src/lib/Kconfig.cbfs_verification

```diff
@@ -26,8 +26,8 @@ config TOCTOU_SAFETY
 	depends on !NO_FMAP_CACHE
 	depends on !NO_CBFS_MCACHE
 	depends on !USE_OPTION_TABLE && !FSP_CAR  # Known to access CBFS before CBMEM init
-	depends on !VBOOT # TODO: can only allow this once vboot fully integrated
-	depends on NO_XIP_EARLY_STAGES
+	depends on ((!VBOOT_CBFS_INTEGRATION && !VBOOT) || (VBOOT_CBFS_INTEGRATION && VBOOT))
```

Comment:
Does `VBOOT_CBFS_INTEGRATION` depend on `VBOOT`? If yes, this can be simplified to `depends on !VBOOT || VBOOT_CBFS_INTEGRATION`.